### PR TITLE
Update Navigation

### LINKED ISSUE #14: Add collapsable menu sub-titles
All “guidelines”, “atoms”, “molecules”, “organisms” and “pages” titles will be collapsed by default to offer a more usable sidenav.

ultimate_toolbox

### PR DESCRIPTION
**Pull request by tonifisler**
Will close #14

### CODE REVIEW

**Comment by Yago** [comment]
There is some merge conflicts..

File: src/components/Item/Item.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+.tlbx-item-code pre {
+  padding: 0.6rem;
+  box-shadow: inset 0 0 0 2px rgba(255,255,255,0.4);
+
+  // Code sample
+  code:last-child {
+    // Hide last empty line (:empty not working, because React)
+    > span:last-of-type {
```
Comment (WengerK) [inline src/components/Item/Item.scss:15]: Be careful, maybe add a hint on the doc about last empty line on every `.twig` file.

File: src/components/Sidebar/Sidebar.js [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-import PropTypes from 'prop-types';
+import { NavLink, withRouter } from 'react-router-dom'
+
+import SidebarItem from '../SidebarItem/SidebarItem';
 
 import './Sidebar.css';
 
 class Sidebar extends Component {
+
+  toggleComponentsList() {
```
Comment (WengerK) [inline src/components/Sidebar/Sidebar.js]: I think it's some legacy code.

File: src/components/SidebarItem/SidebarItem.scss [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  cursor: pointer;
+  text-transform: capitalize;
+  font-size: 1rem;
+
+  &:hover {
+    color: $link-hover-color;
+  }
+
+  &:after {
+    content: "\25BE";
```
Comment (WengerK) [inline src/components/SidebarItem/SidebarItem.scss]: remove it (`content: ''`), use the border-x below.

**Review by WengerK — DISMISSED** [review]
Need just one change. Nice job !

File: src/components/Item/Item.scss (hunk 1 above)
Comment (tonifisler) [reply src/components/Item/Item.scss:15]: We'll do that. See https://github.com/frontend/toolbox/issues/10

**Pull request merged**